packages/pocket-ic/tests/tests.rs

```diff
@@ -1354,6 +1355,7 @@ fn subnet_metrics() {
     assert!((1 << 16) < metrics.canister_state_bytes && metrics.canister_state_bytes < (1 << 17));
 }
 
+#[cfg(unix)]
```

Comment:
Note for reviewers: this test fails on Windows because localhost subdomains (`http://<canister-id>.localhost`) are not supported on Windows.